packages/launchpad/src/components/code/FileRow.cy.tsx

```diff
@@ -16,9 +15,7 @@ export default defineConfig({
   },
 })`
 
-faker.seed(1)
-
-const description = faker.hacker.phrase()
+const description = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.'
```

Comment:
This was regenerating each time - causing Percy snapshots to show as new. We want static text here for snapshot comparison.